toolbox-reference_5b7bdc1d0428631d7a8a2fcc.md

```diff
@@ -229,6 +234,9 @@ Examples:
 Restores an archive with given key.
 In case of a cache hit, archive is retrieved and available at its original
 path in the job environment.
+Cached directory (or file) is restored on the current path, 
```

Comment:
I would say "The cache directory..."

Reply:
🤔 Tnx, but `cache directory` sounds a bit vogue to me. 

With `cached directory` I wanted to stress out that it is related to a specific archive stored by a specific key.

I'm worried that `cache directory` can give the impression that it is the whole cache directory for project, while in this case it is just one archive.

What do you think, would `An archive is restored on the current path` make more sense? cc @mactsouk @darkofabijan

Reply:
Maybe use `Each archive` instead of `An archive`.

Reply:
🤔 I'm still in doubt, thing is that `cache restore` restores only one archive when it is called.

Reply:
So, what about saying:
`Each archive, either directory or single file, is restored in the current path`.
Does this make more sense?

Reply:
Description of cache states that an archive can be a single file or directory. So, I decided to go with `each archive in this example`.

```
cache tool uses key-path pairs for managing cached archives. An archive can be a single file or a directory.
```